main/src/ca/uwaterloo/flix/tools/pkg/MavenPackageManager.scala

```diff
@@ -30,6 +30,7 @@ import java.nio.file.{Files, Path, Paths}
 
 object MavenPackageManager {
 
+  val folderName = "cache"
```

Comment:
private val?

Reply:
I am accessing the name of the folder from Bootstrap, so it cannot be private